setup-dependencies.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
```

Comment:
sh is a common factor, we don't use anything special from bash (nor do we really need to) so might as well just use sh